defaults/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Here is the right place to put the role's input variables.
+# This file also serves as a documentation for such a variables.
+
+# The actual kernel parameters to set, in tuned format.
+# This will be passed directly to the kernel_settings module.
+# See the documentation for the kernel_settings module for the
+# format and fields of this dict object.
+# Example:
+# kernel_settings_parameters:
+#   sysctl:
+#     fs.epoll.max_user_watches: 785592
+#   sysfs:
+#     /sys/fs/selinux/avc/cache_threshold: 256
+kernel_settings_parameters: {}
+# These are operations to apply to the existing config,
+# primarily for the purpose of deleting parameters no
+# longer needed (to have them reset to their kernel defaults),
+# or to replace them with new values specified in
+# kernel_settings_parameters
+# Example - to delete the sysctl parameters kernel.threads-max and fs.file-max,
+# and to remove the entire sysfs section:
+# kernel_settings_operations:
+#   sysctl:
+#   - {"del": "kernel.threads-max"}
+#   - {"del": "fs.file-max"}
+#   sysfs: del
+kernel_settings_operations: {}
+# If force is true, completely wipe out whatever the current settings
+# are and replace them with kernel_settings_parameters
+kernel_settings_force: false
```

Comment:
I know this has a huge bikeshed potential, but I don't think "force" is an appropriate name. To me, "force" sounds like "override some checks", not "wipe everything that's there".

Reply:
FYI the selinux role uses `_purge` variables for that - check the docs. But we were not sure whether it is actually a good idea....

Reply:
using `purge`